installer/create-database.php

```diff
@@ -381,7 +381,8 @@ function populateDatabase($oDB)
             'same_default' =>  "integer NOT NULL default '0'",
             'relevance' =>  "text",
             'question_theme_name' => "string(150) NULL",
-            'modulename' =>  "string(255) NULL"
+            'modulename' =>  "string(255) NULL",
+            'same_script' => "integer NOT NULL default '0'",
```

Comment:
same here

Reply:
Look at line 381 (3 lines above). It is written the same way. I believe it works either way.